app/models/activity.rb

```diff
@@ -0,0 +1,15 @@
+# Activity model for customisation & custom methods
+class Activity < PublicActivity::Activity
+  has_many :notifications, inverse_of: :activity, dependent: :destroy
+
+  # Translate the key of activity to path
+  #   The path will be /ClassName/notifications/activity_type/activity_feed
+  #
+  # @return [String]
+  def key_to_path
```

Comment:
how about <code>template_path</code>, <code>key_to_path</code> is a bit hard to understand